### PR TITLE
Move src\dirs.props properties into the root

### PR DESCRIPTION
**Pull request by mmitche**
Avoids issues where setting Configuration to anything will cause the build to fail because of checks in the Microsoft common targets.

### CODE REVIEW

**Comment by mmitche** [comment]
Fixes #406

File: src/dirs.proj [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" InitialTargets="RestoreAllPackages;VerifyBuildTools" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="dir.props" />
+  <Import Project="..\dir.props" />
```
Comment (weshaggard) [inline src/dirs.proj]: For completeness I would still keep the dirs.props file here even if all it does is import the root one. It at least allows us to add more specific stuff in the future without updating any other projects.

**Comment by weshaggard** [comment]
Can you help me understand which projects get broken by this? I would expect all the *.csproj file to actually work.

**Comment by mmitche** [comment]
All the csproj's work just fine, but you cannot build Release (or explicitly Debug for that matter) from the build.proj

**Pull request merged**